modules/jaeger-rn-new-features.adoc

```diff
@@ -2,27 +2,19 @@
 Module included in the following assemblies:
 - rhbjaeger-release-notes.adoc
 ////
+////
+Feature – Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.
+Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.
+Result – If changed, describe the current user experience.
+////
 
 [id="jaeger-rn-new-features_{context}"]
+== New features {ProductName} 2.0.0
 
+* This release of {ProductName} adds support for using an "external" Elasticsearch cluster to store tracing data, that is, an Elasticsearch instance not installed and created by the Elasticsearch Operator.
 
-== New features {ProductName} 1.17.4
-
-This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
-
-== New features {ProductName} 1.17.3
-
-This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+* This release adds autoscaling support for the Jaeger Collector and Ingester.
 
-== New features {ProductName} 1.17.2
-
-This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
-
-== New features {ProductName} 1.17.1
-////
-Feature – Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.
-Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.  There may not have been a 'problem' previously, but system behaviour may have changed.
-Result – If changed, describe the current user experience
-////
+* This release enabled support for services or applications running outside of an OpenShift cluster to be able to report tracing data to Jaeger running within the OpenShift cluster.
 
-This release of {ProductName} adds support for installing Jaeger as a standalone solution, rather than as a component of Red Hat OpenShift Service Mesh.
+* This release includes a Technology Preview release of clients and infrastructure for exporting tracing data to Jaeger based on the link:https://opentelemetry.io/[OpenTelemetry framework].
```

Comment:
Should this be listed here, if defined in the separate tech preview doc (with a warning). The warning about production usage is missing here.

Reply:
Also the tech preview is associated with Jaeger 2.0, whereas the other features above are associated with Jaeger 1.20.0

Reply:
@objectiser does that mean that if the user installs Jaeger 1.20.0 they will not have access to the OTEL Technology Preview?  I'm a little confused by your second comment.

Reply:
@JStickler Correct - the 1.20.0 release is a GA with no tech preview features. The OTel support is only included in the 2.0 Tech Preview 1 release.

Reply:
@objectiser that is not at all what I was expecting.  Technology Previews are usually included as part of a GA release.  I'm not even sure how to document that the tech preview is a completely separate release.  I'm assuming that installing 2.0 would mean that your entire Jaeger would be technology preview and thus unsupported.

Reply:
@JStickler Service Mesh was a tech preview before 1.0 was released. I agree that sometimes features within a release can be tech preview, but equally a new major release can have tech previews, alphas or betas. It has been discussed as two separate releases.

Correct, 2.0 is completely tech preview and unsupported.